style.css

```diff
@@ -0,0 +1,460 @@
+* {
```

Comment:
How're you making your icons show/work? I had an @import file in mine to make my left-icons show